src/main/java/com/openlattice/shuttle/BuilderCallback.java

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Maybe Ez Kotlinify
unsure about translating annotations to Kotlin 🤔

Reply:
Seems like Kotlin doesn't like `@FunctionalInterface`

Reply:
https://medium.com/androiddevelopers/from-functional-java-to-functioning-kotlin-a4874a4a7a5